packages/hypercontroller/src/jwt.ts

```diff
@@ -59,11 +59,13 @@ const authWithJWT = ({
   jwtOptions,
   loader,
   bearer,
+  authCookieName,
 }: {
   secret: string
   loader: any
   jwtOptions?: any
   bearer?: any
+  authCookieName: string
```

Comment:
did you want to say "if authCookieName is set, the feature is active, if it is not set, the feature is disabled"?

Reply:
@jondot 
Not exactly, the feature is kind of always active since we have a default cookie name that the user can work with, but it is not the first priority of the authentication flow.
The code assumes that the user would have sent a token via one of the two available ways (bearer token or cookie).
Therefore if there is no bearer token, there is no cookie with the given name (default or custom) - the user gets 401 (see the test `fails auth`, last use case)